include/streams.h

```diff
@@ -21,11 +21,11 @@ namespace Php {
 /**
  *  Define the out and err objects
  */
-extern thread_local PHPCPP_EXPORT std::ostream out;
-extern thread_local PHPCPP_EXPORT std::ostream error;
-extern thread_local PHPCPP_EXPORT std::ostream notice;
-extern thread_local PHPCPP_EXPORT std::ostream warning;
-extern thread_local PHPCPP_EXPORT std::ostream deprecated;
+extern thread_local std::ostream out;
+extern thread_local std::ostream error;
+extern thread_local std::ostream notice;
+extern thread_local std::ostream warning;
+extern thread_local std::ostream deprecated;
```

Comment:
IMO, this change is wrong. It needs a profound solution for non GNU compatible compiler. Basically, it still needs a DSO export, but should encapsulate thread safe pieces. I'd allow myself also to reference https://github.com/CopernicaMarketingSoftware/PHP-CPP/issues/90#issuecomment-312434334 . Anyway, it doesn't belong to the PR about static enablement, as it would be controlled by the header where PHPCPP_EXPORT would evaluate to empty already. Right now, the DLL build is simply broken, anyway.

Thanks.